### PR TITLE
[usecase-wrt] Add test for PAC

### PR DESCRIPTION
**Pull request by haoxli**
https://crosswalk-project.org/jira/browse/XWALK-5094
Crosswalk network requests fail on corporate network that
 uses proxy auto-config (PAC) file.

Impacted tests(approved): new 1, update 0, delete 0
Unit test platform: Crosswalk Project for Android 17.45.424.0
Unit test result summary: pass 0, fail 1, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-5101

### CODE REVIEW

File: usecase/usecase-wrt-android-tests/samples/PAC/index.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+</div>
+<div class="content">
+  <h4>Test Steps:</h4>
+  <ol>
+    <li>Connect to Corporate network wifi.</li>
+    <li>Long press on network and click on "Modify network".</li>
+    <li>Check "Advanced options".</li>
+    <li>From proxy drop down select "Proxy Auto-Config".</li>
+    <li>Enter the PAC URL and click save.</li>
+    <li>Visit Baidu and Googel.</li>
```
Comment (Honry) [inline usecase/usecase-wrt-android-tests/samples/PAC/index.html]: 's/Googel/Google'

File: usecase/usecase-wrt-android-tests/samples/PAC/index.html [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+<script src="../../js/tests.js"></script>
+
+<div id="header">
+  <h3 id="main_page_title"></h3>
+</div>
+<div class="content">
+  <h4>Test Steps:</h4>
+  <ol>
+    <li>Connect to Corporate network wifi.</li>
+    <li>Long press on network and click on "Modify network".</li>
```
Comment (Honry) [inline usecase/usecase-wrt-android-tests/samples/PAC/index.html]: Why need to long press? Is this setting on Android system level?

File: usecase/usecase-wrt-android-tests/samples/PAC/index.html [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    <li>Check "Advanced options".</li>
+    <li>From proxy drop down select "Proxy Auto-Config".</li>
+    <li>Enter the PAC URL and click save.</li>
```
Comment (Honry) [inline usecase/usecase-wrt-android-tests/samples/PAC/index.html]: Use which PAC url? Shall we provide the url for tester?

File: usecase/usecase-wrt-android-tests/steps/PAC/step.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+var step = '<font class="fontSize">'
+            +'<p>Purpose:</p>'
+            +'  <p>Verifies that crosswalk network requests corporate network that uses proxy auto-config (PAC).</p>'
+            +'<p>Pre-condition:</p>'
+            +'<ul>'
+            +'  <li>Android 5+ device(PAC support was added in Lollipop).</li>'
+            +'  <li>Network wifi which limited to access google.</li>'
```
Comment (Honry) [inline usecase/usecase-wrt-android-tests/steps/PAC/step.js]: 's/which/which is'

File: usecase/usecase-wrt-android-tests/samples/PAC/index.html (hunk 3 above)
Comment (haoxli) [inline usecase/usecase-wrt-android-tests/samples/PAC/index.html]: This PAC need user to find available one by himself, we provide ones will not be stable.

File: usecase/usecase-wrt-android-tests/samples/PAC/index.html (hunk 2 above)
Comment (haoxli) [inline usecase/usecase-wrt-android-tests/samples/PAC/index.html]: Need to modify the connected wifi, long press will pop up a menu for modify the proxy of the wifi.

Comment (Honry) [inline usecase/usecase-wrt-android-tests/samples/PAC/index.html]: Is this operation suitable for all android 5+ devices? If no, we'd better use a common description.

Comment (haoxli) [inline usecase/usecase-wrt-android-tests/samples/PAC/index.html]: suitable for all android devices

File: usecase/usecase-wrt-android-tests/samples/PAC/index.html (hunk 3 above)
Comment (haoxli) [inline usecase/usecase-wrt-android-tests/samples/PAC/index.html]: It's a auto config script for proxy, i have given a sample in https://crosswalk-project.org/jira/browse/XWALK-5101

**Comment by Honry** [comment]
LGTM

**Pull request merged**